<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTypeTransform_win</class>
 <widget class="QDialog" name="VideoTypeTransform_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="step_1_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1、选择视频文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_1_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选定视频文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_1_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="step_2_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2、选择输出文件夹</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_2_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选定输出文件夹</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_2_line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="step_3_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3、输入输出文件格式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_4_line">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="step_4_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4、输入输出文件名(不填默认相同名字)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="step_4_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="step_5_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>开始转换</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>261</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="step_3_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="step_3_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
